hummingbot/connector/derivative/phemex_perpetual/phemex_perpetual_constants.py

```diff
@@ -6,8 +6,6 @@
 EXCHANGE_NAME = "phemex_perpetual"
 MAX_ORDER_ID_LEN = 40
 
-HB_PARTNER_ID = "HBOT"
```

Comment:
this shouldn't be removed

Reply:
o.0 ... I did not intend to remove this.